### DISCUSSION TITLE
Enhance Channels view to show collapsible list of source and destination connectors like Dashboard view does.

### DISCUSSION THREAD

**Discussion by JeffBenedict** [nextgenhealthcare/connect#5838] [Feature/Enhancement Ideas]
It would be nice if the Channel view showed a collapsible list of source and destination connectors with enabled/disabled status similar to how the Dashboard shows collapsible lists of deployed channels. 

Reason being is that many times in the process of developing channels, being able to document the thought process and seeing visually what is connected to what in order to document and get approval, w/o actually deploying the channels would be beneficial.

![image](https://github.com/nextgenhealthcare/connect/assets/106398018/6d388693-c061-46ae-85d7-259cd142a5ab)
![image](https://github.com/nextgenhealthcare/connect/assets/106398018/a7cce11b-3cee-4783-866f-2334753a2d69)

**Comment by pacmano1** [comment]
I think you mean "show a collapsibile list of source and destination connectors in the Channels tab".  That would be helpful.

**Reply by pacmano1** [reply]
I would add this to the issues/feature section, not discussions.

**Reply by JeffBenedict** [reply]
Thank you.  I've updated original posting with better wording.  And created an issue from this discussion.

**Reply by pacmano1** [reply]
https://github.com/nextgenhealthcare/connect/issues/5839 created
